fleetctl/fleetctl.go

```diff
@@ -496,6 +532,50 @@ func getUnitFromFile(file string) (*unit.UnitFile, error) {
 	return unit.NewUnitFile(string(out))
 }
 
+func getUnitInstanceInfo(name string) (*unit.UnitNameInfo, error) {
```

Comment:
Let's revert this tot he previous logic which just inlined this at the callsite - then we'll get the old error message back which is a little more direct, rather than wrapping like on https://github.com/coreos/fleet/pull/1433/files#diff-197bb752bf0f52567eb6b0023ea65e40R503